src/main/java/carracing/service/CarRacing.java

```diff
@@ -11,7 +11,7 @@
     public CarRacing() {}
 
     public void ready(String carNames, int tryCount) {
-        cars = new Cars(carNames);
+        cars = new Cars(carNames.split(","));
```

Comment:
```suggestion
        cars = new Cars(carNames);
```
이러헥 carNames를 넘겼을 때도 정상적으로 생성할 수 있도록 Cars를 구성해보면 어떨까요?

Reply:
CarRacing이 Service 패키지에 속해있네요. 하는 역할을 보면 사실 Service 보단 Domain에 가깝습니다. 그래서 Domain에 통합하는게 어떨까요? 그럴 경우 테스트 코드도 같이 작성해줘야겠죠!? 😁